homeassistant/components/renson/strings.json

```
@@ -151,5 +151,100 @@
         "name": "Bypass level"
       }
     }
+  },
+  "selector": {
+    "fan_setting": {
+      "options": {
+        "off": "[%key:common::state::off%]",
+        "level1": "[%key:component::renson::entity::sensor::ventilation_level::state::level1%]",
+        "level2": "[%key:component::renson::entity::sensor::ventilation_level::state::level2%]",
+        "level3": "[%key:component::renson::entity::sensor::ventilation_level::state::level3%]",
+        "level4": "[%key:component::renson::entity::sensor::ventilation_level::state::level4%]",
+        "breeze": "[%key:component::renson::entity::sensor::ventilation_level::state::breeze%]",
+        "holiday": "Holiday"
+      }
+    }
+  },
+  "services": {
+    "set_timer_level": {
+      "name": "Set timer",
+      "description": "Set the ventilation timer",
+      "fields": {
+        "timer_level": {
+          "name": "Level",
+          "description": "Level setting"
+        },
+        "time": {
+          "name": "Time",
+          "description": "Time of the timer (0 will disable the timer)"
+        }
+      }
+    },
+    "set_breeze": {
+      "name": "Set breeze",
+      "description": "Set the breeze function of the ventilation system",
+      "fields": {
+        "breeze_level": {
+          "name": "Level",
```

```suggestion
          "name": "[%key:component::renson::services::set_timer_level::fields::timer_level::name%]",
```

Please reuse existing keys as much as possible to reduce the translation overhead.